yamux/src/connection.rs

```diff
@@ -540,12 +543,12 @@ impl<T: AsyncRead + AsyncWrite + Unpin> Active<T> {
     }
 
     fn on_drop_stream(&mut self, id: StreamId) {
```

Comment:
```suggestion
    fn on_drop_stream(&mut self, stream_id: StreamId) {
```

Would be a bit easier to read and remove the need for `let stream_id = id;` below. I know, not directly related, thus feel free to ignore.